_security-plugin/configuration/generate-certificates.md

```diff
@@ -75,20 +75,31 @@ openssl req -new -key admin-key.pem -out admin.csr
 
 Follow the prompts to fill in the details. You don't need to specify a challenge password. As noted in the [OpenSSL Cookbook](https://www.feistyduck.com/books/openssl-cookbook/){:target='\_blank'}, "Having a challenge password does not increase the security of the CSR in any way."
 
-Finally, generate the certificate itself:
+If you generate TLS certificates and have enabled hostname verification by setting `plugins.security.ssl.transport.enforce_hostname_verification` to `true` (default), be sure to specify a common name (CN) for each certificate signing request (CSR) that matches the corresponding DNS A record of the intended node.
+
+If you want to use the same node certificate on all nodes (not recommended), set hostname verification to `false`. For more information, see [Configure TLS certificates]({{site.url}}{{site.baseurl}}/security-plugin/configuration/tls#advanced-hostname-verification-and-dns-lookup).
```

Comment:
Not sure about recommending hostname verification to false. @cliu123 care to comment if this looks okay from security perspective?

Reply:
Yeah I am curious about this as well.  It was already existing in the docs.  The setting is described at https://opensearch.org/docs/latest/security-plugin/configuration/tls#advanced-hostname-verification-and-dns-lookup

Perhaps as an alternative I could reword line 80 to read something along the lines of:

```
To learn more about settings related to TLS certificates in OpenSearch, see [Configure TLS certificates]({{site.url}}{{site.baseurl}}/security-plugin/configuration/tls#advanced-hostname-verification-and-dns-lookup).
```

Reply:
> Not sure about recommending hostname verification to false. @cliu123 care to comment if this looks okay from security perspective?

It's fine to recommend hostname verification to false. Actually this is the only choice in this case, otherwise, verification would fail and error would be thrown. This is not a recommanded use case, which has already been noted as `(not recommended)`. So the descritpion makes sense to me.